<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapesAndLayersFontManagerHelperDialog</class>
 <widget class="QDialog" name="ShapesAndLayersFontManagerHelperDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Font Manager Helper</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Manual Import</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="9,1,2">
         <item>
          <widget class="QLabel" name="fontPathLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Select a font</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="fontPathBtn">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="folder">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="importFontBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Import</string>
           </property>
           <property name="icon">
            <iconset theme="document-import"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="loadedFontsLabel">
           <property name="text">
            <string>System Fonts: 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reloadSystemFontsBtn">
           <property name="text">
            <string>Reload All System Fonts</string>
           </property>
           <property name="icon">
            <iconset theme="view-refresh"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Auto Import</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="watchTempFontDirLabel">
           <property name="text">
            <string>Temporary Font Dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="watchTempFontDirBtn">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="folder"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="toggleWatchTempFontDirBtn">
           <property name="text">
            <string>Start Watching</string>
           </property>
           <property name="icon">
            <iconset theme="media-playback-start"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="watchSystemFontDirBtn">
         <property name="text">
          <string>Start Watching System Fonts For Changes</string>
         </property>
         <property name="icon">
          <iconset theme="media-playback-start"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="fontList"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShapesAndLayersFontManagerHelperDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShapesAndLayersFontManagerHelperDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
